src/lib/component/SaveAnnotationDialog/bind.js

```diff
@@ -17,12 +18,14 @@ export default function (
   // Save to the URL.
   delegate(element, '.textae-editor__save-dialog__url-text', 'keyup', (e) => {
```

Comment:
```suggestion
  delegate(element, '.textae-editor__save-dialog__url-text', 'keyup', async (e) => {
```

もしかして、ここにasyncをいれたら、このあとで呼び出される一連のメソッドをasyncに出来ないでしょうか？
もしそうであれば、Promiseとasyncが混ざった書き方をasyncに統一出来そうです。

Reply:
完全に理解しているわけではないのですが、delegateに渡すコールバック関数をasyncにせずともこの後の処理をasyncに統一して問題なく動作したので、delegateに渡す関数はasyncとせずに後の処理をasync化しました。
https://github.com/pubannotation/textae/pull/159/commits/f7e114c9fd663a8d22aaee6a3dbe194e9887d056